manifests/pipelines/aiedge-e2e.pipeline.yaml

```diff
@@ -310,7 +319,7 @@ spec:
   - name: test-model-rest-svc
     params:
     - name: model-name
-      value: $(params.model-name)
+      value: $(tasks.sanitise-model-name.results.output-string)
     - name: model-version
       value: $(params.model-version)
```

Comment:
As @grdryn suggested [here](https://github.com/opendatahub-io/ai-edge/pull/280#pullrequestreview-2215325557) we can parameterize this with the service name rather than model-name and model-version. This will make the task more configurable and users can choose their own deployment and service naming. Now they are basically forced to use `$(tasks.sanitise-model-name.results.output-string)-$(params.model-version)` if they want to use this testing task.

Reply:
Yep makes sense will change